### PR TITLE
Check if commands in screenshot test have indices > 1

### PR DESCRIPTION
**Pull request by hliatis**
Bug: b/171891634

### CODE REVIEW

File: cmd/gapit/commands.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func (verb *commandsVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 
 	return traverseCommandTree(ctx, client, tree.Root, func(n *service.CommandTreeNode, prefix string) error {
 		if verb.OnlyExecutedDraws {
-			if n.Group != "" || n.NumChildren > 0 {
+			if n.Group != "" || n.NumChildren > 0 || len(n.Commands.First().Indices) == 1 {
```
Comment (hevrard) [inline cmd/gapit/commands.go:107]: Could you add a comment to clarify why this additional check is necessary? (maybe explaining that some queueSubmit may pass the two other checks yet not be relevant for OnlyExecutedDraws)

File: cmd/gapit/screenshot.go [hunk 2]
```diff
@@ -272,7 +272,7 @@ func (verb *screenshotVerb) executedDrawCommands(ctx context.Context, capture *p
 
 	var allDrawCommands []*path.Command
 	traverseCommandTree(ctx, client, tree.Root, func(n *service.CommandTreeNode, prefix string) error {
-		if n.Group != "" || n.NumChildren > 0 {
+		if n.Group != "" || n.NumChildren > 0 || len(n.Commands.First().Indices) == 1 {
```
Comment (hevrard) [inline cmd/gapit/screenshot.go:275]: idem

**Review by hevrard — APPROVED** [review]
LGTM, thanks!

**Review by pmuetschard — APPROVED** [review]
This is for queue submits with 0 submit infos? yikes.

**Pull request merged**